### PR TITLE
feat: add global config for nydus-snapshotter

### PR DESCRIPTION
**Pull request by gaius-qi**

### CODE REVIEW

File: charts/nydus-snapshotter/values.yaml [hunk 1]
```diff
@@ -1,13 +1,27 @@
 # nydus-snapshotter Helm Chart Values
+#
+global:
```
Comment (imeoer) [inline charts/nydus-snapshotter/values.yaml:3]: Perhaps comment that what's case will use the global here.

File: charts/nydus-snapshotter/values.yaml [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
 name: nydus-snapshotter
-# -- Image repository
-image: ghcr.io/containerd/nydus-snapshotter
-# -- Image tag
-tag: v0.9.0
-# -- Image pull policy
-pullPolicy: Always
+image:
+  # -- Image registry.
+  registry: ghcr.io
```
Comment (imeoer) [inline charts/nydus-snapshotter/values.yaml:15]: Is this necessary when we have `global.imageRegistry` ?

**Review by imeoer — APPROVED** [review]
LGTM! by offline discussion.

**Comment by adamqqqplay** [comment]
Thanks!

**Pull request merged**